packages/modalConnector/package.json

```diff
@@ -0,0 +1,76 @@
+{
+    "name": "@ripio/sdk-modal-connector",
+    "version": "0.0.0",
```

Comment:
v0.0.0?

Reply:
yes, semantic release will do the job to update it